src/components/RightPanelContents.tsx

```diff
@@ -70,27 +65,28 @@ const RightPanelContents = (props: AllProps) => {
         darkMode={props.darkMode}
       />
       <SearchDiv darkMode={props.darkMode}>
-        <StyledInput
-          type="text"
-          value={searchQuery}
-          onChange={handleChange}
-          onKeyDown={handleKeyDown}
-          darkMode={props.darkMode}
-        />{" "}
-        <StyledSvg
-          onClick={handleSubmit}
-          darkMode={props.darkMode}
-          viewBox="0 0 16 16"
-        >
-          <path
-            fill-rule="evenodd"
-            d="M10.442 10.442a1 1 0 0 1 1.415 0l3.85 3.85a1 1 0 0 1-1.414 1.415l-3.85-3.85a1 1 0 0 1 0-1.415z"
-          />
-          <path
-            fill-rule="evenodd"
-            d="M6.5 12a5.5 5.5 0 1 0 0-11 5.5 5.5 0 0 0 0 11zM13 6.5a6.5 6.5 0 1 1-13 0 6.5 6.5 0 0 1 13 0z"
-          />
-        </StyledSvg>
+        <form onSubmit={handleSubmit}>
+          <StyledInput
+            type="text"
+            value={searchQuery}
+            onChange={handleChange}
+            darkMode={props.darkMode}
+          />{" "}
+          <StyledSvg
```

Comment:
Could you wrap this in a `<button type="submit>` and get rid of the onClick? That's more in line with accessibility guidelines.

Generally if something is going to be a button, you should implement it as a styled `<button` of some sort so you can get all the default controls that buttons get for keyboard navigation and screen readers.

It'd also be good to add an [aria-label](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute) attribute or a [title attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/title) saying this is a "Search Button".

Might be good to do an accessibility review at some point, too.

Reply:
Okay! I think the latest commit resolves those issues. I like the idea of an accessibility review!